src/Runner/Linter.php

```diff
@@ -82,13 +89,18 @@ private function fix(iterable $finder, Ruleset $ruleset): void
 
         foreach ($finder as $file) {
             $filePath = $file->getPathname();
-            $success = $fixer->fixFile($filePath);
+            $success = true;
+            if ($this->cacheManager->needFixing($filePath, file_get_contents($filePath))) {
+                $success = $fixer->fixFile($filePath);
+            }
```

Comment:
```suggestion
            if (!$this->cacheManager->needFixing($filePath, file_get_contents($filePath))) {
                continue;
            }
            $success = $fixer->fixFile($filePath);
```
no ?

Reply:
Yes. I went a bit further as phpstan complained about possible `false` value being passed to `needFixing` param #2.

```php
            $contents = file_get_contents($filePath);
            if (false === $contents || !$this->cacheManager->needFixing($filePath, $contents)) {
                continue;
            }

            if (!$fixer->fixFile($filePath)) {
                throw new Exception(sprintf('Cannot fix the file "%s".', $filePath));
            }
```

I got rid of `$success` in the process.